Jenkinsfile

```diff
@@ -38,17 +36,19 @@ pipeline {
             sh 'git tag -a $BRANCH_NAME-build-$BUILD_NUMBER -m "Jenkins"'
             sh 'git push https://${GIT_USR}:${GIT_PSW}@github.com/yonadev/yona-app-ios.git --tags'
           }
-          
         }
-        
         archiveArtifacts 'Yona/BuildOutput/**/*.ipa'
       }
       post {
         always {
           junit 'Yona/BuildOutput/**/*.xml'
-          
         }
-        
+        success {
+          slackSend color: 'good', channel: '#dev', message: "iOS app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} succeeded"
```

Comment:
I want to know difference between `${BUILD_NUMBER}` and `${env.BUILD_NUMBER}`.

Reply:
The first one refers to a Groovy variable named ``BUILD_NUMBER``, the second one to a map entry ``BUILD_NUMBER`` in the Groovy variable ``env``, which holds the environment variables.

Reply:
Ok Thanks.